### DISCUSSION TITLE
Sandbox IDE --> doesn't display .html when private ?

### DISCUSSION THREAD

**Discussion by FlorianCourtaigne** [codesandbox/codesandbox-client#6377] [Q&A]
Hi, 
I have different .html on the mode private in sandbox with a few .html file.
The principal one is index.html and when I try to display an another one it doesn't work.
I tried it on Chrome, Firefox and Edge.
Thank you for your help

**Comment by JamesACS** [comment]
Hey @flowweb92 - Could you please link us to the Sandbox in question? 

What are you trying to accomplish?

**Reply by FlorianCourtaigne** [reply]
Hi James,

I'm quiet new in web developement and I don't know very much Codesandbox.
I was just wondering if, in the case of multiple .html pages on the same project, when you select one of the .html file of, it would automatically display it on the right side of the screen ? Or if you have to do it with integreted links ?

Here is a very basic project with multiple .html pages.
https://codesandbox.io/s/multiple-pages-forked-eb61y?file=/index.html

I would expect that, when you select "gallery" for example, it would display the content of gallery.html directly on the right side of the screen. But maybe it is not a possibility of Codesandbox ?

Thank you,

**Reply by FlorianCourtaigne** [reply]
@JamesACS

**Comment by JamesACS** [comment]
Ah I understand @flowweb92 - Unfortunately, this isn't possible as CodeSandbox will always render the index page first. You'll need to build in routing to the other pages in order to view them in the preview

**Reply by FlorianCourtaigne** [reply]
Alright thank you for your answer.
